### ISSUE TITLE
minDate & maxDate not working on iOS?

### ISSUE THREAD

**Issue by msqar** [thodubois/react-native-woodpicker#16]
I'm trying to use minDate and maxDate but doesn't seem to work on iOS.

This is how my Picker looks:

```
<DatePicker
        onDateChange={this.handleDatePicker}
        value={this.state.pet.dateOfBirth}
        title={i18n.t('dateOfBirth')}
        placeholder={this.handleDatePickerPlaceholder()}
        placeholderStyle={scopedStyles.datePickerText}
        iosPickerMode="date"
        androidPickerMode="spinner"
        locale="en"
        minDate={ () => {
              var date = new Date();
              date.setFullYear(date.getFullYear() - 3);
              return date;
         }}
         maxDate={new Date()}
/>
```

Am I doing it fine?

After checking the code, it looks like the `minDate` and `maxDate` are being handled by `handleAndroidDatePicker()` function and is only used for the Android Picker instance... nothing at all on the iOS picker. Seems it's not supported by this lib yet?

**Comment by msqar** [comment]
Okay, fixed it. Would be nice to create a PR for this. It works awesome now.

These are the changes on DatePicker.js for this to work:

```
class IOSDateWoodPicker extends PureComponent {
  render() {
    const {
      containerStyle,
      renderInput,
      renderDoneBar,
      showPicker,
      togglePicker,
      fadeAnim,
      pickedDate,
      onDateChange,
      minDate,   // < ------
      maxDate,   // < ------
      locale,
      iosPickerMode
    } = this.props;
    return (
      <View style={containerStyle}>
        {renderInput()}
        <Modal
          visible={showPicker}
          transparent
          animationType="slide"
          supportedOrientations={["portrait", "landscape"]}
        >
          <TouchableOpacity style={styles.blurTouchable} onPress={togglePicker}>
            <Animated.View
              style={{
                flex: 1,
                backgroundColor: "#000000",
                opacity: fadeAnim
              }}
            />
          </TouchableOpacity>
          {renderDoneBar()}
          <View style={styles.iosPickerContainerStyle}>
            <DatePickerIOS
              mode="date"
              date={pickedDate}
              onDateChange={onDateChange}
              locale={locale}
              minimumDate={minDate} // < ------
              maximumDate={maxDate} // < ------
              mode={iosPickerMode}
            />
          </View>
        </Modal>
      </View>
    );
  }
}
```

Then this one:

```
 render() {
    const {
      containerStyle,
      locale,
      iosPickerMode,
      placeholder,
      placeholderStyle,
      style,
      isNullable,
      minDate, // < ------
      maxDate, // < ------
      title,
      doneText
    } = this.props;
    const { showPicker, fadeAnim, pickedDate } = this.state;
    return isIOS ? (
      <IOSDateWoodPicker
        containerStyle={containerStyle}
        locale={locale}
        iosPickerMode={iosPickerMode}
        showPicker={showPicker}
        fadeAnim={fadeAnim}
        maxDate={maxDate} // < ------
        minDate={minDate} // < ------
        pickedDate={pickedDate}
        togglePicker={this.togglePicker}
        onDateChange={this.onDateChange}
        renderInput={() =>
          this.renderInput(
            this.togglePicker,
            style,
            placeholder,
            placeholderStyle,
            isNullable
          )
        }
```

**Comment by msqar** [comment]
Created a PR for this missing piece.
https://github.com/thodubois/react-native-woodpicker/pull/17

**Comment by thodubois** [comment]
Great! I check this today.

**Comment by msqar** [comment]
@thodubois thanks :) let me know so i update to the version with the fix. I need it haha.

**Comment by thodubois** [comment]
Hello @msqar,  your fix is available on 0.0.16 =)
Thanks for your contribution!
